### PR TITLE
Update category-list.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Devdocs improvement with example of url_key and url_path attribute with categoryList query.

## Purpose of this pull request

This pull request (PR) describe the more live example of the url_key and url_path attribute with categoryList query. Example for the ids and name attribute available. This will be helpful for the user to get all the available example demo to learn categoryList query.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/graphql/queries/category-list.html
https://devdocs.magento.com/guides/v2.4/graphql/queries/category-list.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/category-list.md [hunk 1]
```diff
@@ -276,6 +276,86 @@ The following query returns breadcrumb information about categories that have th
 }
 ```
 
+### Return the categoryList by url_key filters
+
+The following query returns information about category by url_key filters. You have to pass url_key value without suffix with `eq` or `in` keyword. Get the categoryList response from the `Gear` category.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/category-list.md:281]:
```suggestion
The following query returns information about categories by `url_key` filters. You have to pass the `url_key` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear` category.
```

File: src/guides/v2.3/graphql/queries/category-list.md [hunk 2]
```diff
@@ -276,6 +276,86 @@ The following query returns breadcrumb information about categories that have th
...
+        "children_count": "3"
+      }
+    ]
+  }
+}
+```
+
+### Return the categoryList by url_path filters
+
+The following query returns information about category by url_path filters. You have to pass url_path value without suffix with `eq` or `in` keyword. Get the categoryList response from the `Gear -> Bags` category.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/category-list.md:321]:
```suggestion
The following query returns information about categories by `url_path` filters. You have to pass the `url_path` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear -> Bags` category.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice additions.- just some grammar corrections.

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated the suggested changes. Thank you.

File: src/guides/v2.3/graphql/queries/category-list.md [hunk 3]
```diff
@@ -276,6 +276,86 @@ The following query returns breadcrumb information about categories that have th
 }
 ```
 
+### Return the categoryList by url_key filters
+
+The following query returns information about categories by `url_key` filters. You have to pass the `url_key` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear` category.
```
Comment (keharper) [inline src/guides/v2.3/graphql/queries/category-list.md:281]:
```suggestion
The following query returns information about the Gear category using a `url_key` filter. You must pass the `url_key` value without a suffix and specify either the `eq` or `in` keyword.
```

File: src/guides/v2.3/graphql/queries/category-list.md [hunk 4]
```diff
@@ -276,6 +276,86 @@ The following query returns breadcrumb information about categories that have th
...
+        "children_count": "3"
+      }
+    ]
+  }
+}
+```
+
+### Return the categoryList by url_path filters
+
+The following query returns information about categories by `url_path` filters. You have to pass the `url_path` value without a suffix and with either the `eq` or `in` keywords. Get the categoryList response from the `Gear -> Bags` category.
```
Comment (keharper) [inline src/guides/v2.3/graphql/queries/category-list.md:321]:
```suggestion
The following query returns information about the Gear > Bags` category using the `url_path` filter. Do not specify a suffix in the `url_path` value. The `url_path` filter accepts either the `eq` or `in` keyword.
```

**Review by keharper — CHANGES_REQUESTED** [review]

**Comment by RakeshJesadiya** [comment]
@keharper I have update the docs as per your suggestion. Thank you.

**Review by keharper — APPROVED** [review]

**Pull request merged**